### PR TITLE
Consistently use same default model

### PR DESCRIPTION
**Pull request by svlandeg**
## Description
Use the same default model for the `llm` factory and the `llm_task` factories, so that these are entirely equivalent:
```
llm = nlp.add_pipe("llm", config={"task": {"@llm_tasks": "spacy.NER.v3"}})
```
and
```
llm = nlp.add_pipe("llm_ner")
```

### Corresponding documentation PR

NA

### Types of change
fix

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran all tests in `tests` and `usage_examples/tests`, and all new and existing tests passed. This includes
  - all external tests (i. e. `pytest` ran with `--external`)
  - all tests requiring a GPU (i. e. `pytest` ran with `--gpu`)
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy_llm/pipeline/llm.py [hunk 1]
```diff
@@ -21,6 +21,20 @@
 logger = logging.getLogger("spacy_llm")
 logger.addHandler(logging.NullHandler())
 
+DEFAULT_MODEL_CONFIG = {
+    "@llm_models": "spacy.GPT-3-5.v1",
```
Comment (svlandeg) [inline spacy_llm/pipeline/llm.py:25]: Should be v2 when https://github.com/explosion/spacy-llm/pull/236/files is merged

Comment (rmitsch) [inline spacy_llm/pipeline/llm.py:25]:
```suggestion
    "@llm_models": "spacy.GPT-3-5.v2",
```

File: spacy_llm/tests/tasks/test_ner.py [hunk 2]
```diff
@@ -248,6 +248,35 @@ def test_ner_predict(cfg_str, text, gold_ents, request):
...
+    doc = nlp(text)
+
+    assert len(doc.ents) == len(gold_ents)
+    for pred_ent, gold_ent in zip(doc.ents, gold_ents):
+        assert (
+            gold_ent[0] in pred_ent.text
+        )  # occassionally, the LLM predicts "in Ireland" instead of just "Ireland"
+        assert pred_ent.label_ in gold_ent[1].split("|")
+
+
```
Comment (rmitsch) [inline spacy_llm/tests/tasks/test_ner.py:279]: Do we need a new test for this - can't we just parametrize the other `predict` test? Overall I don't care strongly, so go ahead and merge if you prefer it this way.

**Review by rmitsch — COMMENTED** [review]
LGTM, just one minor comment.

File: spacy_llm/pipeline/llm.py (hunk 1 above)
Comment (svlandeg) [reply spacy_llm/pipeline/llm.py:25]: Like I said - this requires https://github.com/explosion/spacy-llm/pull/236 to be merged first, or this PR's CI will fail

File: spacy_llm/tests/tasks/test_ner.py (hunk 2 above)
Comment (svlandeg) [reply spacy_llm/tests/tasks/test_ner.py:279]: I did consider that, but the other `predict` test relies on a `config` parameter which this new one explicitely shouldn't.

**Comment by rmitsch** [comment]
Should succeed after merging in `develop`.

**Pull request merged**